paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentMethodVerticalLayoutUI.kt

```diff
@@ -0,0 +1,36 @@
+package com.stripe.android.paymentsheet.ui
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.stripe.android.lpmfoundations.luxe.SupportedPaymentMethod
+import com.stripe.android.uicore.image.StripeImageLoader
+
+@Composable
+internal fun PaymentMethodVerticalLayoutUI(
```

Comment:
Created this new layout for the list of both saved PMs + new PMs (using the existing NewPaymentMethodVerticalLayoutUI). Once there is a saved PM displayed in this layout, it will have a "View more" button and clicking that button should call the onViewMorePaymentMethods function that is included here

Reply:
Yeah, we will probably keep doing this for a while. I'm hoping to refactor some stuff in this realm too. But I'm on board with this as iterative progress!